pymc3/model.py

```diff
@@ -434,6 +439,23 @@ def __init__(self, name='', model=None):
             self.potentials = treelist()
             self.missing_values = treelist()
 
+        if theano_config is None:
+            theano_config = {'compute_test_value': 'raise'}
+        self._theano_config = theano.configparser.change_flags(**theano_config)
+        # We need to enter the theano config, since variables could be
+        # created in the init of subclasses.
+        self._theano_config.__enter__()
```

Comment:
Hm I see the problem now. Yes, It can be done in `__new__`([here](https://github.com/pymc-devs/pymc3/blob/master/pymc3/model.py#L418)), that's the best place for that.